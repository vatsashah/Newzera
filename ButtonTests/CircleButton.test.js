// Importing required library
import React from 'react'
import { shallow, mount } from 'enzyme'

// Importing component to be tested
import CircleEllipse from '../../../Components/Buttons/circleEllipse'

describe('CircleEllipse', () => {
  it('Circular Button should render correctly', () => {
    const component = shallow(<CircleEllipse />)
    expect(component).toMatchSnapshot()
  })
  it('Circular Button should be disabled', () => {
    const wrapper = mount(<CircleEllipse disabled />).props()
    expect(wrapper.disabled).toBe(true)
  })
  it('Circular Button should be filled', () => {
    const wrapper = mount(<CircleEllipse filled />).props()
    expect(wrapper.filled).toBe(true)
  })
})
