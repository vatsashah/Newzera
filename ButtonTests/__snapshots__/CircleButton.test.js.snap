// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CircleEllipse Circular Button should render correctly 1`] = `
<div
  className="default"
>
  <img
    alt="Greater than symbol"
    className="arrowFix"
    src="RightArrowColoured.png"
  />
</div>
`;
